hangman.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, secret_movie=None, max_guesses=10) -> None:
         self.wrong_guesses = 0
         self.max_guesses = max_guesses
         self.already_guessed = set()
-        self.correct_letter = set()
+        self.correct_letters = set()
```

Comment:
given this set holds multiple letters, I think this is a more accurate variable name